tests/unit/helpers/remotecontrol/ExportResponsesStataXmlTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace ls\tests;
+
+/**
+ * Tests for ExportSTATAxml.
+ */
+class ExportResponsesStataXmlTest extends BaseTest
+{
+    private static $username = null;
+    private static $password = null;
+    private static $pluginName = 'ExportSTATAxml';
+    private static $plugin;
+
+    public static function setUpBeforeClass(): void
+    {
+        parent::setUpBeforeClass();
+
+        \Yii::import('application.helpers.remotecontrol.remotecontrol_handle', true);
+        \Yii::import('application.helpers.viewHelper', true);
+        \Yii::import('application.libraries.BigData', true);
+
+        // Get a handle to the plugin being tested
+        self::$plugin = App()->getPluginManager()->loadPlugin(self::$pluginName);
+
+        // Activate it, if not already
+        self::installAndActivatePlugin(self::$pluginName);
+
+        // Import survey
+        $filename = self::$surveysFolder . '/survey_export_responses_with_tokens.lsa';
+        self::importSurvey($filename);
+    }
+
+    /**
+     * Get responses in stataxml format.
+     */
+    public function testGetResponses()
+    {
+        $sessionKey = $this->handler->get_session_key($this->getUsername(), $this->getPassword());
+
+        $result = $this->handler->export_responses($sessionKey, self::$surveyId, 'stataxml');
+        $this->assertNotNull($result);
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        self::$testSurvey->delete();
```

Comment:
If i don't make error , it's already in parent

Reply:
Strange : https://github.com/LimeSurvey/LimeSurvey/blob/804319af283f3be4c01c676279caeae8027d99f7/tests/TestBaseClass.php#L133

Reply:
Have just updated it. Let me see how it goes now.